### ISSUE TITLE
Add new fingerprint ({modelID: "WSD500A", manufacturer name: " _TZ3000_saiqcn0y"})

### ISSUE THREAD

**Issue by misha1183** [Koenkk/zigbee-herdsman-converters#6515]
Add new fingerprint ({modelID: "WSD500A", manufacturer name: " _TZ3000_saiqcn0y"})

![photo_2023-11-20_14-56-19](https://github.com/Koenkk/zigbee-herdsman-converters/assets/87187097/b3b62dc8-8a19-4af7-afd3-697dfbbd8ebf)

**Comment by Koenkk** [comment]
Could you provide the data/database.db entry of this device?

**Comment by misha1183** [comment]
How can i do it? Sorry, I don't know.

**Comment by Koenkk** [comment]
It's already there; https://github.com/Koenkk/zigbee-herdsman-converters/blob/a93da0b31a4cc29a601150220c0460ceb1a1b440/src/devices/tuya.ts#L1959

**Comment by misha1183** [comment]
Yes, but there is another device on photo. It isn't problem? Thank you.

**Comment by Koenkk** [comment]
Fixed!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
